airflow/models/xcom.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
```

Comment:
an earlier commit separated models/__init__.py objects into their own separate files. The commits I cherrypicked only added this one line, but the merge conflict meant I had to create this file, which I think will not work.

Incorporating all the model extractions is a much more involved set of changes, so we may want to rethink our strategy here..